NDB.Covid19/NDB.Covid19.iOS/Views/MessagePage/MessagePageViewController.cs

```diff
@@ -67,7 +67,7 @@ public override void ViewWillDisappear(bool animated)
 
         private async void SetStyling()
         {
-            StyleUtil.InitLabelWithSpacing(Label, StyleUtil.FontType.FontBold, MESSAGES_HEADER, 1.14, 22, 34);
+            StyleUtil.InitLabelWithSpacing(Label, StyleUtil.FontType.FontBold, MESSAGES_HEADER, 0.9, 22, 34);
```

Comment:
0.9 ? :)

Reply:
The descender on the G in the heading is cut if it's set any higher :/

Reply:
I can see some pixels are cut there as well with 0.9. We need to rework it.